mos-platform/nes/neslib/ntsc.s

```diff
@@ -3,9 +3,15 @@
 .include "ntsc.inc"
 .include "neslib.inc"
 
-.section .init.275,"axR",@progbits
+.section .init.330,"axR",@progbits
```

Comment:
We should maintain the property that the neslib initializers are done before `.init.300`; that's when user-defined module level constructors fire, and broadly anything that is accessible in C should work properly from those pre-main constructors, including the NTSC mode.

This is true here and throughout this file; anything that previously occurred before `.init.300` should stay before `.init.300`.

Reply:
looks like i forgot to include `mos-platform/common` when I was searching for existing init ranges.